src/GitLabApiClient/Internal/Http/GitLabApiPagedRequestor.cs

```diff
@@ -1,8 +1,9 @@
-﻿using System;
+using System;
```

Comment:
I suspect you encoded the file in UTF-8 BOM for Net standard I would avoid it 😅

Reply:
@casz Wired. Normally opened the file in VS 2019 and ran all tests after that 😟 Will see tomorrow what's wrong. In my personal projects I use `.editorconfig` to avoid this.

Reply:
I'll have a quick look in vscode

Reply:
😆 You fixed the UTF-8 BOM to UTF-8. So all good!

Reply:
Editorconfig does say UTF-8 is wanted https://github.com/nmklotas/GitLabApiClient/blob/master/.editorconfig#L19

Reply:
😓 Seems VS and R# respect the `.editorconfig` 😉